Readme.md

```diff
@@ -1,25 +1,15 @@
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [CriteriaPatcherBase.cs](./CS/App_Code/CriteriaPatcherBase.cs) (VB: [CriteriaPatcherBase.vb](./VB/App_Code/CriteriaPatcherBase.vb))
-* [CriteriaVisitor.cs](./CS/App_Code/CriteriaVisitor.cs) (VB: [CriteriaVisitor.vb](./VB/App_Code/CriteriaVisitor.vb))
-* [MyCustomFunctionOperator.cs](./CS/App_Code/MyCustomFunctionOperator.cs) (VB: [MyCustomFunctionOperator.vb](./VB/App_Code/MyCustomFunctionOperator.vb))
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to apply Custom Function Filter Criteria Operator
+# Grid View for ASP.NET Web Forms - How to apply custom function filter criteria operators
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t546944/)**
 <!-- run online end -->
 
+This example demonstrates how to use a [CriteriaPatchBase](https://supportcenter.devexpress.com/ticket/details/t320172/how-to-traverse-through-and-modify-the-criteriaoperator-instances) class descendant to apply a [custom function criteria operator](https://docs.devexpress.com/eXpressAppFramework/113480/filtering/in-list-view/custom-function-criteria-operators).
+
+![Custom function filter criteria operator](customFunctionCriteriaOperator.png)
 
-<p>Some complicated filter cases require visiting every leaf of the filter criteria tree when a <a href="https://documentation.devexpress.com/eXpressAppFramework/113480/Concepts/Filtering/Custom-Function-Criteria-Operators">custom function criteria operator</a> is used. <br>This example shows how it can be implemented via <a href="https://www.devexpress.com/Support/Center/Question/Details/T320172/the-base-implementation-of-the-iclientcriteriavisitor-interface-for-the-criteriaoperator">CriteriaPatchBase's</a> class descendant.<br>The main idea is to operate with ASPxGridView's <a href="https://documentation.devexpress.com/AspNet/DevExpressWebASPxGridBase_FilterExpressiontopic.aspx">FilterExpression</a>:<br>1. If FilterExpression has a custom function operator, it's necessary to find exactly this custom operator and replace its value with an actual one. <br>In the case of Auto Filter Row, this can be done in the <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebASPxGridView_ProcessColumnAutoFiltertopic">ASPxGridView.ProcessColumnAutoFilter</a> event handler as shown below:</p>
+## Overview
 
+Use the grid's [FilterExpression](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.FilterExpression) property to replace a custom function operator with an actual one. For auto filter row, handle the grid's server-side [ProcessColumnAutoFilter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.ProcessColumnAutoFilter) event as follows:
```

Comment:
What do you mean by `with an actual one`? A custom function operator is still an operator, so what is this 'actual one' you mention?

`For auto filter row` Do you mean "For the auto filter row"?